ckanext/cfpb_extrafields/templates/package/resource_read.html

```diff
@@ -38,6 +38,34 @@ <h1 class="page-heading">{{ h.resource_display_name(res) | truncate(50) }}</h1>
 {# moved down below and forced to be on #}
 {% endblock %}
 
+{% block resource_actions %}
+<ul>
+  {% block resource_actions_inner %}
+  {% if h.check_access('package_update', {'id':pkg.id }) %}
+    <li>{% link_for _('Manage'), controller='package', action='resource_edit', id=pkg.name, resource_id=res.id, class_='btn', icon='wrench' %}</li>
+  {% endif %}
+  {% if res.url and h.is_url(res.url) and not " " in res.url and not res.resource_type == "Data Dictionary" %}
```

Comment:
It's one more ugly on a stack of ugly related to urls being required where they shouldn't be. Anyway, let's add `not res.resource_type == "Database"`